object/search.go

```diff
@@ -134,6 +134,7 @@ const (
 	FilterPayloadHomomorphicHash = v2object.FilterHeaderHomomorphicHash
 	FilterParentID               = v2object.FilterHeaderParent
 	FilterSplitID                = v2object.FilterHeaderSplitID
+	FilterFirstObjectID          = v2object.ReservedFilterPrefix + "split.firstID"
```

Comment:
while it's true bout API Go, dont forget to extend the protocol https://github.com/nspcc-dev/neofs-api/blob/5c3535423c564fe63991812cb84d8334db1c553a/object/service.proto#L508-L529

u'll also notice there that all previous declarations follow json tags, so `$Object:split.first` would be closer. But techically there is no hard coupling, `firstID` is also ok to me

Reply:
ok `first` is better